mod/storage/pkg/block/store.go

```diff
@@ -32,9 +32,9 @@ import (
 // KVStore is a simple memory store based implementation that stores metadata of
 // beacon blocks.
 type KVStore[BeaconBlockT BeaconBlock] struct {
-	blockRoots       *lru.Cache[common.Root, math.Slot]
-	executionNumbers *lru.Cache[math.U64, math.Slot]
-	stateRoots       *lru.Cache[common.Root, math.Slot]
+	blockRoots *lru.Cache[common.Root, math.Slot]
+	timestamps *lru.Cache[math.U64, math.Slot]
+	stateRoots *lru.Cache[common.Root, math.Slot]
```

Comment:
@abi87 all of these mappings are actually in the form of some key -> slot value. In this case we store blockRoot -> slot, timestamp -> slot, stateRoot -> slot. Maybe its better to rename these more explicitly, along the lines of  `blockRootsToSlots` or similar

Reply:
Naming as is works fine for me.